utam-compiler/src/main/resources/errors.config.json

```diff
@@ -0,0 +1,59 @@
+{
+  "P001": {
```

Comment:
codes stand for:
- Pxxx means it's general parsing error that can happen for any entity
- UA stands for UtamArgument error
- UE for UtamElement error etc.
Abbreviation is there to easier navigate through codes